react-native-gutenberg-bridge/ios/RNReactNativeGutenbergBridge.swift

```diff
@@ -8,9 +8,16 @@ public class RNReactNativeGutenbergBridge: RCTEventEmitter {
     // MARK: - Messaging methods
 
     @objc
-    func provideToNative_Html(_ html: String, title: String, changed: Bool) {
+    func provideToNative_Html(_ html: String, title: String, changed: Bool, contentInfo: [String:Int]) {
         DispatchQueue.main.async {
-            self.delegate?.gutenbergDidProvideHTML(title: title, html: html, changed: changed)
+            var info: ContentInfo?
+            if  let characters = contentInfo["characterCount"],
+                let words = contentInfo["wordCount"],
+                let paragraphs = contentInfo["paragraphCount"],
+                let blocks = contentInfo["blockCount"] {
+                info = ContentInfo(characterCount: characters, wordCount: words, paragraphCount: paragraphs, blockCount: blocks)
+            }
```

Comment:
Maybe we can do this as a `ContentInfo` struct extension?
In that way we clear up a bit this code.

Maybe we could also `Decodable` but just if it's not more complex (or an overkill) 🤔
What I like from Decodable is that for any change in the Object from JS, we just need to do one change in one place (the definition struct itself).

Reply:
Good suggestion!